ipt_connect/IPTdev/templates/IPTdev/build_tactics.html

```diff
@@ -3,6 +3,14 @@
 {% load i18n %}
 
 {% block content %}
+    <div class="content container">
+        <div class="section">
+            <h1>
+                {% trans 'This is beta feature, use it at your own risk!' %}
```

Comment:
You could add that if a weird behaviour is spotted, feedback to us is appreciated :)

Reply:
Done:
https://github.com/IPTnet/ipt_connect/pull/129/commits/a93f8f1d666e4d5df372abf71a00f1ccf2debe85